_chapters/pt/who-is-this-guide-for.md

```diff
@@ -12,6 +12,26 @@ Esse guia foi feito para desenvolvedores full-stack ou desenvolvedores que desej
 
 Talvez você seja um desenvolvedor backend querendo aprender mais sobre a parte frontend de aplicações Serverless, ou um desenvolvedor frontend que gostaria de aprender mais sobre backend. Esse guia servirá para ambos os casos.
 
-Por hora, apenas vamos abordar o desenvolvimento com JavaScript. Futuramente talvez abordemos outras linguagens e ambientes. Porém, para começar, nós achamos muito benéfico para um desenvolvedor full-stack utilizar apenas uma linguagem (JavaScript) e ambiente (NodeJs) para a construção de uma aplicação completa.
+Pessoalmente, a ideia do Serverless foi uma enorme revelação para nós e isso nos fez criar um guia com qual poderíamos compartilhar o que aprendemos. Você pode saber mais sobre nós [**aqui**]({% link about/index.html %}). E [veja alguns exemplos de pessoas que construíram aplicações com SST clicando aqui]({% link showcase.md %})
 
-Pessoalmente, a ideia do Serverless foi uma enorme revelação para nós e isso nos fez criar um guia com qual poderíamos compartilhar o que aprendemos. Você pode saber mais sobre nós [**aqui**]({% link about/index.html %}). E [veja alguns exemplos de pessoas que construiram aplicações com a stack Serverless]({% link showcase.md %})
+Por hora, apenas vamos abordar o desenvolvimento com JavaScript/TypeScript. Futuramente talvez abordemos outras linguagens e ambientes. Porém, para começar, nós achamos muito benéfico para um desenvolvedor full-stack utilizar apenas uma linguagem (JavaScript) e ambiente (NodeJs) para a construção de uma aplicação completa.
```

Comment:
(JavaScript) at the end of the line was changed to (TypeScript) in the last change in the EN chapter and I think that (NodeJs) can be changed to (Node.js) to be the same as the EN version.